tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -122,7 +122,7 @@
 # Allow the platform to include itself in an iframe
 X_FRAME_OPTIONS = "SAMEORIGIN"
 
-{% set jwt_rsa_key = rsa_import_key(JWT_RSA_PRIVATE_KEY) %}
+{% set jwt_rsa_key | rsa_import_key %}{{ JWT_RSA_PRIVATE_KEY }}{% endset %}
```

Comment:
I don't know what's going on here but I'll trust you on it.

Reply:
This is actually one of the few occurrences where typing highlighted a bug. `rsa_import_key` was considered as a Jinja2 variable, but variables are not expected to be callables. So we moved it to the list of Jinja2 filters instead, but that means that we have to use it slightly differently: https://jinja.palletsprojects.com/en/3.0.x/templates/#block-assignments

Reply:
Got it. Cool!